tests/util/test_check_dependencies.py

```diff
@@ -12,25 +27,25 @@
 
 
 class DummyDistribution(metadata.Distribution):
-    def __init__(self, version: object):
+    def __init__(self, version: str):
```

Comment:
Maybe `version: Optional[str]`?

Reply:
This forces `version` to return `Optional[str]`, which then doesn't match the super-class?

Reply:
Ahh yes, I'd forgotten that I inherited from the proper Distribution when setting up this mocking

Reply:
Yeah, any thought on if I should change this then or just leave it?

Reply:
Oh sorry, let's leave it as is!